distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/ServerCliTests.java

```diff
@@ -611,6 +605,10 @@ public LoadedSecrets load(Environment environment, Terminal terminal) throws Exc
         @Override
         public SecureSettings bootstrap(Environment environment, SecureString password) throws Exception {
             this.bootstrapped = true;
+            // make sure we don't fail in fips mode when we run with an empty password
+            if (password == null || password.isEmpty()) {
```

Comment:
Can we make this special case specific to fips, instead of creating a fallback that shouldn't be applicable to other cases?

Reply:
yeah good point, I'll add that additional check.